docker/README.md

```diff
@@ -5,11 +5,29 @@ The image is built from a versioned Rocker image, with additional R packages and
 It can be built with the following command run in this directory:
 
 ```
-docker buildx build . -t scpca-tools --platform linux/amd64
+docker buildx build . -t scpcatools --platform linux/amd64
 ```
 
-Note that this image does not include RStudio, in attempt to make the image smaller. 
+Note that this image does not include RStudio, in attempt to make the image smaller.
 
 This image is built automatically on every update to `scpcaTools` (`main`) and pushed to `ghcr.io/alexslemonade/scpca-tools:edge`.
 
-Release versions of `scpcaTools` will be tagged as `ghcr.io/alexslemonade/scpca-tools:latest` and tagged by version number.
+Release versions of `scpcaTools` will be tagged as `ghcr.io/alexslemonade/scpcatools:latest` and tagged by version number.
+
+The main image includes all of the recommended packages for `scpcaTools` (aside from those only required for development), as well as the Python `scanpy` and `scvi-tools` packages and dependencies.
+
+There are also smaller images that can be built, which include only the R packages required for `scpcaTools`, with particular additions for specific purposes. These include:
+
+- `scpcatools_slim.Dockerfile`: A smaller image that includes only the required R packages for `scpcaTools`, with no additional packages.
+- `scpcatools_anndata.Dockerfile`: The slim package with the addition of the `zellkonverter` package for reading and writing `AnnData` objects, and the Python `anndata` package.
```

Comment:
The anndata package also includes the scanpy and scvi-tools right?

Reply:
So for right now, it did not, but I could make one that does. Scanpy and scvi add a lot of additional dependencies, so I wasn't sure we would want the whole image there. But maybe that is the way to go, just to reduce the total number of images.

Reply:
I feel like we should have one that has anndata + scvi. We will for sure need that for scpca-nf, and then maybe later down the line we can separate them even further to have just an anndata one. I would keep the number of images minimal at this stage.